core/src/mongosqltranslate.rs

```diff
@@ -222,9 +240,13 @@ impl CheckDriverVersion {
 
 #[derive(Debug, Serialize, Deserialize)]
 pub enum CommandResponse {
+    #[serde(rename = "translation")]
```

Comment:
The library will have to send the translation result back in the key "translation"